.github/workflows/lint.yml

```
@@ -0,0 +1,31 @@
+name: Lint
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [ "main", "dev" ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)